README.md

```diff
@@ -364,6 +364,10 @@ It includes:
 From CLI run:
 
 ```bash
+# build monaco-vscode-api (the demo use it as a local dependency)
+npm ci
```

Comment:
You should also mention that the project does not properly build on Windows. You require Linux (including WSL) or Mac.

Reply:
Good idea! 500285db26ca5ca74021a0aa95ca447d0cf17490